### DISCUSSION TITLE
ScyllaDB support?

### DISCUSSION THREAD

**Discussion by samof76** [Ericsson/ecchronos#752] [General]
Hi.

Does ecchronos support scylladb?

Thanks
Sam

**Comment by tommystendahl** [comment]
Hi,
Thanks for reaching out and showing interest.

From what I understand scylladb uses rest instead of jmx that cassandra uses so unfortunately there is no support for scylladb. SInce we only use cassandra scylla support is not something we will work on.

But I found [scylladb/scylla-jmx](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://github.com/scylladb/scylla-jmx&ved=2ahUKEwjPmL6MtZeJAxXuT0EAHW3FHVwQFnoECA0QAQ&usg=AOvVaw22PZ7nB4322ZDFVUSdmEuY) that loks like it provides a cassandra compatible jmx interface as a bridge to scylladbs rest interface, maybe this can be used but we have no way of testing such a solution. If you do test this yourself it would be interesting to hear about the result and it could be useful for others.
